exporter/exporterhelper/common.go

```diff
@@ -185,9 +184,15 @@ func newBaseExporter(set exporter.CreateSettings, signal component.DataType, req
 		unmarshaler:     unmarshaler,
 		signal:          signal,
 
-		queueSender:   &baseRequestSender{},
-		obsrepSender:  osf(obsReport),
-		retrySender:   &errorLoggingRequestSender{logger: set.Logger},
+		queueSender: &errorLoggingRequestSender{
+			logger:  set.Logger,
+			message: "Exporting failed. Dropping data. Try enabling sending_queue to survive temporary failures.",
+		},
+		obsrepSender: osf(obsReport),
+		retrySender: &errorLoggingRequestSender{
+			logger:  set.Logger,
+			message: "Exporting failed. Dropping data. Try enabling retries to survive temporary failures.",
+		},
```

Comment:
We will be showing these messages in the exporters that don't have an option to enable sending_queue or retries (no `WithQueue`/`WithRetry` used). That would be confusing

Reply:
We need to keep separate behaviors for unavailable or disabled

Reply:
Good point. Let's do that then: if "disabled" we print these messages, if not called at all (a.k.a. not available) we don't print anything.